src/frontend/src/handler/explain.rs

```diff
@@ -99,6 +102,25 @@ async fn do_handle_explain(
                         )?
                         .0
                     }
+
+
+                    (None, Some(cdc_table)) => {
+                        gen_create_table_plan_for_cdc_source(
+                            context.into(),
+                            cdc_table.source_name.clone(),
+                            name.clone(),
+                            cdc_table.external_table_name.clone(),
+                            columns,
+                            constraints,
+                            ColumnIdGenerator::new_initial(),
+                        )?.0
+                    }
+                    (Some(_), Some(_)) => return Err(ErrorCode::NotSupported(
+                        "Data format and encoding format doesn't apply to table created from a CDC source"
+                            .into(),
+                        "Remove the FORMAT and ENCODE specification".into(),
+                    )
+                        .into()),
```

Comment:
Changes to this file are bug fix for explain of cdc. Previously this logic is only located in `handle_create_table`.

This logic can be further unified with that in `handle_create_table`.

Further refactoring should:
1. Provide an `options` struct to simplify argument passing.
2. unify the match functionality between both, to avoid this bug from happening again.

Reply:
https://github.com/risingwavelabs/risingwave/issues/13363

Reply:
Refactoring will not be done here. It is out of scope for purposes of this PR. Only just fix so we can add a planner test.